data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -321,11 +411,12 @@ private StructLikeSet selectColumns(StructLikeSet rows, String... columns) {
     return set;
   }
 
-  private StructLikeSet rowSetWithoutIds(int... idsToRemove) {
+  private StructLikeSet rowSetWithoutIds(Table iTable, List<Record> recordList, int... idsToRemove) {
```

Comment:
Since you're passing the table in, can you make this a `static` method? You may also be able to use the name `table` instead of `iTable` if it is `static`.

Reply:
A helpful suggestion, it works, thanks!